libraries/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -354,41 +354,41 @@ static bool _validateSubscription( bool awsIotMqttMode,
     /* Check for a valid QoS and callback function when subscribing. */
     if( operation == IOT_MQTT_SUBSCRIBE )
     {
-        status = _validateQos( pSubscription->qos );
+        if( pSubscription->callback.function == NULL )
```

Comment:
Due to this change, line 364 could be changed to an `else`

Reply:
changed